### PR TITLE
ICS 26 Routing module minimal implementation for MVP

### PR DESCRIPTION
**Pull request by adizere**
Primarily closes: #232 
Should also handle the last bits to close: #159 

## Description

______

For contributor use:

- [ ] Unit tests written
- [ ] Added test to CI if applicable 
- [ ] Updated CHANGELOG_PENDING.md
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: modules/src/lib.rs [hunk 1]
```diff
@@ -15,9 +15,11 @@
 //!
 //! - ICS 02: Client
 //! - ICS 03: Connection
+//! - ICS 04: Channel
 //! - ICS 07: Tendermint Client
 //! - ICS 23: Vector Commitment Scheme
```
Comment (ancazamfir) [inline modules/src/lib.rs:20]: add ICS 20?

File: modules/src/ics26_routing/handler.rs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            let handler_output =
+                ics3_msg_dispatcher(ctx, msg).map_err(|e| Kind::HandlerRaisedError.context(e))?;
+
+            ctx.store_connection_result(handler_output.result)
+                .map_err(|e| Kind::KeeperRaisedError.context(e))?;
+        } // TODO: add dispatchers for ICS4 and others.
+    };
+
+    Ok(())
+}
```
Comment (ancazamfir) [inline modules/src/ics26_routing/handler.rs:44]: maybe add some tests using the mock context? these would be similar to the ones in the ics03 dispatch.

**Review by ancazamfir — COMMENTED** [review]
Looks great! A couple of minor comments.

File: modules/src/lib.rs (hunk 1 above)
Comment (adizere) [reply modules/src/lib.rs:20]: I hesitated as it was in a very minimal state. But good point, it's better to document it here regardless.

**Review by brapse — APPROVED** [review]
Looks like a good first step 👍

**Pull request merged**